src/components/MainSideBar/index.js

```diff
@@ -61,35 +74,121 @@ const TagContainer = () => {
         alignItems: 'center',
         alignSelf: 'center',
         justifyContent: 'flex-end',
+        textAlign: 'center',
       }}
     >
       <Tag to="/settings/experimental">
-        <Trans i18nKey="common.experimentalFeature" />
+        <img src={i('experimental.svg')} alt="" draggable="false" width={16} height={16} />
```

Comment:
Why is this an `img` and not an icon like we do for all other icons? I suggest you convert the svg to a react component with something like https://www.smooth-code.com/open-source/svgr/playground/